apps/frontend/package.json

```diff
@@ -104,6 +104,7 @@
     "sass": "~1.32.6",
     "sass-loader": "^10.0.3",
     "search-query-parser": "^1.5.5",
```

Comment:
if we can get rid of unnecessary dependencies, let's do so

Reply:
search-query-parser is now removed